Mathlib/RingTheory/DedekindDomain/FiniteAdeleRing.lean

```diff
@@ -109,8 +109,7 @@ def Coe.addMonoidHom : AddMonoidHom (R_hat R K) (K_hat R K) where
     -- Porting note (https://github.com/leanprover-community/mathlib4/issues/11041): was `ext v`
     refine funext fun v => ?_
     simp only [coe_apply, Pi.add_apply, Subring.coe_add]
-    -- Porting note: added
-    rw [Pi.add_apply, Pi.add_apply, Subring.coe_add]
+    rfl
```

Comment:
I guess this is fine, but please double-check.

Reply:
Yeah, the two `rfl`s in this file seem ok. They could be turned into new `@[simp]` lemmas too, but I don't know if that's worth the effort.